src/pathways/pathway/pathways.service.ts

```
@@ -2439,7 +2445,9 @@ export class PathwaysService {
               '{firstName}': user.firstName,
               '{lastName}': pathwayName,
               '{courseName}': pathwayName,
-              '{programName}': record.pathway.subtype ?? '',
+              '{programName}': record.pathway.subtype
```

ESLint

[error] Replace `'{programName}'` with `"{programName}"`

(prettier/prettier)